### ISSUE TITLE
[sticker pack] Clyde Stickers

### ISSUE THREAD

**Issue by kelseydieterich** [michaellee/iOS-iMessage-Stickers-Icon-Sketch-Template#2]
https://itunes.apple.com/us/app/clyde-stickers/id1292150783?mt=8

www.kelseydieterich.com

Thanks, Michael!

**Comment by michaellee** [comment]
Hey hey @kelseydieterich thanks for letting me know, I've put it in the [readme.md](https://github.com/michaellee/iOS10-iMessage-Stickers-Icon-Sketch-Template). Clyde looks like a great set! Is Clyde an original character of yours? I'm also curious how you found this template?

Cheers and really enjoyed seeing your work on your website!

**Comment by kelseydieterich** [comment]
Thank you for your compliments on Clyde and the website! He is an original character. I found your template in Adelyn Tam's Medium post about making sticker packs. Her post and your template really helped me out. 

Thanks again! :)

**Comment by michaellee** [comment]
Awesome! Glad it helped you :) Do you have a Twitter account? Would love to share with others.

**Comment by kelseydieterich** [comment]
I do! That would be awesome. It's @kelseycd3 

Also, if you'd like here's a code to download Clyde - XFWLNEWTFPF7

**Comment by michaellee** [comment]
THANK YOU SO MUCH KELSEY!!! Shared it on my Twitter account [@michaelsoolee](https://twitter.com/michaelsoolee). I hope you get loads of downloads and hope you have a great weekend!

**Comment by michaellee** [comment]
Oh by the way @kelseydieterich is there anything that I could've done better to improve the sticker making process with this template?

**Comment by kelseydieterich** [comment]
Nothing needs improvement, it's well organized and easy to navigate, which is key. One thing I did do was add an extra art board for the app store icon, which is 1024x1024pt, so I could have all my icon builds in one place. This icon comes in later when submitting the app in iTunes Connect. I'm thinking that size was added to Apple's requirements after the template was built.

Thanks again for the Twitter posts!
